### DISCUSSION TITLE
Emit doesn't work but wire:click="myFunction" works

### DISCUSSION THREAD

**Discussion by christoferd** [livewire/livewire#5519] [1. Help]
I don't know enough about inner workings of Livewire to say if this is a bug, or caused by a limitation (e.g. limit of nested components)

**Scenario**
I have a Livewire component (Dropdown), that is being used within another livewire component (Editor), where Editor is opened within a modal, only when required. There is only one instance of Editor on the page. Editor is only opened by another livewire event, it opens when they click on a calendar entry.

**Issue:**
Only clicking on "This Works" works. That click will cause an ajax call, all others do nothing and do not cause any errors.

```html
<button wire:click="$emit('optionClicked', 'aaaa')">aaaa</button> (not work)

<button wire:click="optionClicked('bbbbb')">This Works</button> (works)

<button @click="Livewire.emit('optionClicked', 'cccc')">cccc</button>  (not work)

<button onclick="window.livewire.emit('optionClicked', 'dddd')">dddd</button>  (not work)
```

Any ideas as to why this is?

**Comment by Martleby** [comment]
Are you listening for the events on your server side component with 

```protected $listeners = [....]``` ?
